### ISSUE TITLE
False positive with Objects.toString

### ISSUE THREAD

**Issue by cushon** [typetools/checker-framework#4889]
After https://github.com/typetools/jdk/commit/8b504384bd6f609282f30fa2a5a19c00f8aecaa5, the nullness checker reports the following false positive. Should the first parameter of `Objects.toString` be annotated `@Nullable` instead of `@PolyNull`?

```java
import java.util.Objects;
import org.checkerframework.checker.nullness.qual.Nullable;

class T {
  void f(@Nullable String s) {
    Objects.toString(s, "").toString();
  }
}
```

```
$ ./checker/bin/javac -processor Nullness T.java
...
T.java:6: error: [dereference.of.nullable] dereference of possibly-null reference Objects.toString(s, "")
    Objects.toString(s, "").toString();
```

**Comment by mernst** [comment]
Liam, you are right;  I should have added `@PolyNull` only where there was no annotation before.
Thank you for the bug report!
